<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>321</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>99999</width>
    <height>99999</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>System Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="versionlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Sw Version :</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Device Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="deviceIdlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>115</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Device Version :</string>
   </property>
  </widget>
  <widget class="QLabel" name="fwversionlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>165</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>FW Version :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>System Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>150</width>
     <height>179</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/YouMengInfo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Thin Client</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>115</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>MC5181R</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>YMAAP201706261635</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>165</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>YMAUB201704203144</string>
   </property>
  </widget>
  <widget class="QLabel" name="snnumlabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>215</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>E-mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>215</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>support@unicomlife.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="snnumlabel_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Website:</string>
   </property>
  </widget>
  <widget class="QLabel" name="devname_label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>www.unicomlife.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="IPlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Device IP :</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
